chapter-03/recipe-10/c-example/hwserver.c

```diff
@@ -1,41 +0,0 @@
-//  Hello World server
```

Comment:
This is the correct version of `hwserver.c`, since it will work on Windows and UNIX.